addon/components/primitives/frost-list-item-end-point.js

```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember'
```

Comment:
Endpoint is a very "ciena" concept should we really have something like this in a "generic" repo?